modules/claims_api/spec/lib/claims_api/v2/disability_compensation_pdf_mapper_spec.rb

```diff
@@ -282,7 +282,7 @@
 
         expect(start_date).to eq('03-1985')
         expect(no_date).to eq(false)
-        expect(treatment_details).to eq('Traumatic Brain Injury - Center One, Decatur, GA')
+        expect(treatment_details).to eq('Traumatic Brain Injury, Post Traumatic Stress Disorder (PTSD) Combat - Mental Disorders, Cancer - Musculoskeletal - Elbow - Center One, Decatur, GA') # rubocop:disable Layout/LineLength
```

Comment:
same as above. https://github.com/department-of-veterans-affairs/vets-api/pull/13181/files#r1263938486